test/cms/page/diversions_test.exs

```diff
@@ -0,0 +1,50 @@
+defmodule CMS.Page.DiversionsTest do
```

Comment:
📓 This is really just a copy of the `/page/basic_test.exs`. For tests related to the CMS, we have to use the static JSON responses until we rewrite all of the CMS tests.